test/specs/AutoRegisterDataProvideSpec.js

```diff
@@ -0,0 +1,21 @@
+describe("Auto register data-provide Tests", function() {
+  it("checks that the autoregister Slider was automatically registerd", function() {
```

Comment:
I actually left that out intentionally, because the Slider is activated on page-load, not inside the test.
So if I destroy it inside the test, the result of other tests might change with the order of the tests, because destroying an auto-slider would change global state.